src/main/java/org/apache/ibatis/migration/Environment.java

```diff
@@ -233,6 +251,26 @@ public String getPassword() {
     return password;
   }
 
+  public String getMigrationLogDriverPath() {
+    return driverPath;
```

Comment:
Shouldn't this be migrationLogDriverPath ? (and below as well?)

Reply:
Right 🤦  
I would've been quite surprised when testing this :D